presto-rcfile/src/main/java/com/facebook/presto/rcfile/GenericRcFileCompressor.java

```diff
@@ -23,12 +23,12 @@
 
 import static java.util.Objects.requireNonNull;
 
-public class AircompressorCompressor
+public class GenericRcFileCompressor
```

Comment:
I'd call this `HadoopCompressor` since it is using the Hadoop `CompressionCodec`